index.html

```diff
@@ -161,6 +172,40 @@ <h3>Skills</h3>
                 </ul>
             </div>
         </section>
+        <section class="contact-form">
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/xayvgeok" method="post">
+                <ul>
+                    <li>
+                        <label for="name"></label>
+                        <input type="text" name="full-name" id="name" placeholder="Full name" required>
+                    </li>
+                    <li>
+                        <label for="mail"></label>
+                        <input type="email" name="email" id="mail" placeholder="Email address" required>
+                    </li>
+                    <li>
+                        <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here"
+                            required></textarea>
+                    </li>
+                    <li>
+                        <button type="submit" name="send" value="send" class="button-style">Get in touch</button>
```

Comment:
- Please, kindly add the interactions on the button to indicate the active, hover, enabled and disabled state to improve the user experience below:

![](https://user-images.githubusercontent.com/59397159/177452937-fe738a71-0f3f-43bd-8d8b-4a1734a8ce39.png)

Reply:
@zieeco thanks a lot for mention that i totally forgot about that i just implemented it you can have a look if you want thanks again